build/valgrind/openssl.supp

```diff
@@ -11,6 +11,6 @@
    fun:_ZN2mk8libevent6Poller4loopEv
    fun:_ZN2mk7Reactor23loop_with_initial_eventESt8functionIFvvEE
    fun:_ZN2mk23loop_with_initial_eventESt8functionIFvvEENS_3VarINS_7ReactorEEE
-   fun:_ZL31____C_A_T_C_H____T_E_S_T____418v
+   fun:_ZL31____C_A_T_C_H____T_E_S_T____*
```

Comment:
I decided to use this wildcard to match all tests having this valgrind problem.
The first implementation used another specific suppression inside this file, but I find this solution more easy and clear.